com.sap.cloud.lm.sl.cf.core/src/test/java/com/sap/cloud/lm/sl/cf/core/util/ApplicationURITest.java

```diff
@@ -24,9 +34,100 @@
     @MethodSource("testParameters")
     public void testGetHostDomainPath(String uri, String expectedHost, String expectedDomain, String expectedPath) {
         ApplicationURI applicationURI = new ApplicationURI(uri);
-        Assertions.assertEquals(expectedDomain, applicationURI.getDomain());
-        Assertions.assertEquals(expectedDomain, ApplicationURI.getDomainFromURI(uri));
-        Assertions.assertEquals(expectedHost, applicationURI.getHost());
+        assertParametersAreValid(expectedDomain, applicationURI.getDomain(), expectedDomain, ApplicationURI.getDomainFromURI(uri),
```

Comment:
This method has way too many arguments

Reply:
+ the getDomain() instance method and getDomainFromURI() static method do the same thing as here. They create an appUri object and extract its domain.
Why are these two duplicate values needed?

Reply:
The method is removed, but getDomainFromURI() is a convenient method which is used in the code.